index.html

```diff
@@ -4240,47 +4240,54 @@ <h2>HTTP</h2>
         </section>
 
         <section class="informative">
-          <h2>JSON Processing</h2>
-
+          <h2>JSON-LD Processing</h2>
           <p>
 While the media types describing conforming documents defined in this
-specification always express JSON-LD, JSON-LD processing is not required to be
-performed, since JSON-LD is JSON. Some scenarios where processing a
-<a>verifiable credential</a> or a <a>verifiable presentation</a> as JSON is
+specification always express JSON-LD, RDF processing is not required to be
+performed, since JSON-LD is a concrete RDF syntax as described in [RDF11-CONCEPTS]. 
+Hence, a JSON-LD document is both an RDF document and a JSON document and correspondingly represents an instance of an RDF data model.
+See <a href="JSON-LD11##relationship-to-rdf">Relationship to RDF</a> for more details.
```

Comment:
The notion of "RDF processing" is not defined, and it can mean many things. Better not go there. Besides, it took me some time to understand what (I think) you meant in this section... What about a rewrite of the whole paragraph along the lines of:

> While the media types describing conforming documents defined in this specification all express JSON-LD, developers should be aware that JSON-LD is a serialization of the abstract RDF Model [[RDF11-CONCEPTS]]. Hence, a JSON-LD document is both a JSON document and the representation of an abstract RDF Dataset. See <a href="JSON-LD11#relationship-to-rdf">Relationship to RDF</a> for more details. 

Also, in view of what you write later, _I have the feeling_ that you have in mind two type of "processing":

- "RDF Processing" when you transform the JSON-LD into RDF using the JSON-LD rules
- "JSON-LD Processing" when you operate on the JSON level only.

if that is correct, this may be the place to define these, because otherwise the text below is very unclear. That being said, I do not like the two terms; for me, "JSON-LD Processing" means to perform the processing steps defined in the [JSON-LD 1.1 Processing Algorithms and API](https://www.w3.org/TR/json-ld11-api/) specification, and this seems to contradict this.

Reply:
^ this is correct, my goal is to basically say this:

1. You need to care about JSON-LD
2. You can care about it and ignore the context, you will have a bad time.
3. You can care about it and pay attention to the context, and W3C or any other 3rd party context host can cause you to have a bad time (by changing the context after you commit to it).
4. In the case that all contexts are immutable and always available, you can still have equivocation, but at least its limited to differences between RDF and JSON, not RDF and JSON-LD (where there will be NO equivocation in this very special case).

Reply:
> You can care about it and ignore the context, you will have a bad time.

No, this is not a thing. You never "ignore" `@context`. You can hard-code your application to only work with some specific contexts, that's fine. But you don't "ignore" `@context`, that is not at all the same thing. Perhaps this is part of a core misunderstanding here?

The comments about context immutability are a different kind of issue. We can have text here that says: "if you're going to hard-code to specific contexts, they must be immutable in order to guarantee the same interpretation as someone who doesn't hard-code". That is a simple fact and encouraging the use of immutable contexts for VCs is a good thing, IMO.